package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "yarn",
   "installationMethod": "unknown",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
this is unnecessary, the core team will bump the patch version when they are ready to release it

Reply:
I will revert this. Thank you.